framework/src/media/FocusRequest.cpp

```diff
@@ -16,21 +16,20 @@
  *
  ******************************************************************/
 
-#include <sstream>
 #include <media/FocusRequest.h>
 
 namespace media {
 std::string FocusRequest::getId()
 {
-	return mId;
+	return std::to_string(mStreamInfo->id);
```

Comment:
I think now, getStreamInfo is correct, instead of getId.

Reply:
I'll fix it with another PR.